src/components/form/submission-confirmation.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from "../ui/button";
+import Image from "next/image";
+import Link from "next/link";
+
+export function SubmissionConfirmation() {
+  return (
+    <div className="flex flex-col border-2">
+      <div className="z-10 -mb-4 h-4 w-full rounded-t-xl bg-gradient-to-r from-rose-400 via-yellow-400 to-blue-600" />
+      <div className="flex h-[80vh] w-full flex-col items-center justify-center space-y-6 rounded-xl bg-white px-8 py-32 text-cooper-blue-600">
+        <Image
+          src="./svg/verified.svg"
+          width={125}
+          height={125}
+          alt="Verified"
+        />
+        <div className="text-5xl font-bold">
+          Your review has been submitted!
+        </div>
+        <div className="max-w-3xl text-center text-2xl">
+          Thank you for taking the time to submit a co-op review! Your feedback
+          is highly appreciated.
+        </div>
+        <div className="flex justify-between space-x-8">
+          <Link href="/">
+            <Button>Return to home </Button>
```

Comment:
Since this button doesn't have a border like the one next to it, it is a few pixels smaller. I think you could either add a border to this button that has the same color so it gets bigger, or use the tailwind classes `outline outline-[color]` as I don't think those add size like border does.